kolibri/content/api.py

```diff
@@ -4,221 +4,173 @@
 """
 from functools import wraps
 
-from django.core.files import File as DjFile
 from django.db.models import Q
 from kolibri.content import models as KolibriContent
 from kolibri.content.utils import validate
+from .constants import content_kinds
 
 """ContentDB API methods"""
 
 def can_get_content_with_id(func):
     """
-    decorator function for returning ContentMetadata object when giving content id.
+    decorator function for returning ContentNode object when giving content id.
```

Comment:
This decorator will return a contentnode object if you pass the corresponding id. If you pass a contnetnode object directly, this decorator will do nothing, just return what you pass in.
I see the naming is bit confusing, should I change it, any suggestions?